ci/benchmark_parser.py

```diff
@@ -95,19 +95,25 @@ def recursive_parse(directory, walk_subdirs=False, name_suffix="", compute_throu
 
             for stat_name, value in parse_estimate_file(subdir).items():
                 test_name_parts = list(filter(None, [test_name, stat_name, name_suffix]))
+
+                if "throughput" in test_name:
+                    bench_type = "throughput"
+                else:
+                    bench_type = "latency"
+
                 result_values.append(
                     _create_point(
                         value,
                         "_".join(test_name_parts),
                         bench_class,
-                        "latency",
+                        bench_type,
                         operator,
                         params,
                         display_name=display_name
                     )
                 )
 
-                if stat_name == "mean" and compute_throughput:
+                if stat_name == "mean" and "throughput" not in test_name and compute_throughput:
```

Comment:
is the throughput not in test name required ? that would be weird if compute_throughput is True but we don't compute it ?

Reply:
Oh you're right ! The output from criterion is a latency even if we set up the bench to be parallel. We still have to compute the throughput while parsing.

Reply:
Does not look like it is solved (sorry if it was in the other commit you were working on)